tests/python/rest_api/test_tasks.py

```diff
@@ -2858,3 +2858,97 @@ def test_can_export_and_import_skeleton_tracks_in_coco_format(self):
                 for tes in te.shapes
             ]
         )
+
+    @pytest.mark.parametrize("format_name", ["Datumaro 1.0", "COCO 1.0", "PASCAL VOC 1.1"])
```

Comment:
Probably, it makes sense to add or extend an existing test for the `COCO keypoints` format, as it's also affected.